README.md

```diff
@@ -231,6 +255,23 @@ Description: [Overall description]
 /*************************/
 
 ```
+*Scripts should be clearly organized so others can follow them. 
+*If a directory needs to be set, it should be set at the top of a script. If multiple directories are needed, you can use macros (e.g., “global” in Stata) to name them and reference them throughout. It should not be necessary to change multiple file pathways throughout a script to run it on a different computer. 
+##Commenting
+*Include comments throughout your scripts so others can follow your work and decisions. 
+* Include comments that state "why", not "what". Include comments for all assumptions.
+##Naming & Dates
+* Use descriptive names for all variables, data sets, functions, and macros. 
+*Avoid abbreviations. 
+* Use ISO 8601 dates (YYYY-MM-DD).
+##Assertions and Tests
+* Write assertions and in-line tests. Assertions, things expected to always be true about the code, should be tested in-line. [healthinequality-code](https://github.com/michaelstepner/healthinequality-code/blob/master/code/readme.md#assert-what-youre-expecting-to-be-true) offers some good background. `assert` is useful in Stata and `stopifnot()` is useful in R. 
+* Write tests for final files. Write a test if all numbers should be non-negative. Write a test if values should not exceed $3,000. 
```

Comment:
Check that the value ranges make sense. Spot-check your outliers to confirm that those values are not an error. In some cases, you might need to dig on the internet to see if say a community has the worst rates of air quality that there's some verification of that. Or if homelessness among students spiked that there's some context that could explain that. If not, then check your code to make sure it is doing what you think it is doing. 

Also, check the data quality flag. Look at the distribution of assigned quality.